app/assets/javascripts/discourse/app/components/d-toggle-switch.hbs

```diff
@@ -1,14 +1,14 @@
 <div class="d-toggle-switch">
   <label class="d-toggle-switch--label">
-    {{! template-lint-disable no-unnecessary-concat }}
+    {{! template-lint-disable no-redundant-role }}
```

Comment:
it was (incorrectly I believe) complaining about the use of `role="switch"` on a `<button>`